### DISCUSSION TITLE
How to make a TabBar vertically oriented and left-aligned.

### DISCUSSION THREAD

**Discussion by satyampriyam08** [unoplatform/uno#9319] [Q&A]
I created a new Uno platform app and selected Win Ui as one of the platforms.
I need a tab menu that is vertically oriented and left-aligned on the page.
I used TabBar control from the namespace - xmlns:utu="using:Uno.Toolkit.UI"
I am unable to vertically orient and left-align it.

Need support.

**Comment by francoistanguay** [comment]
Why not use the Navigation with LeftCompactMode? https://docs.microsoft.com/en-us/windows/apps/design/controls/navigationview

Right now, we are not providing a Vertical style for TabBar but should be feasible.

**Comment by kazo0** [comment]
Progress/planning for a Vertical TabBar can be tracked here: https://github.com/unoplatform/uno.toolkit.ui/issues/278
